apps/sms/views/conversation/js/conversation.js

```diff
@@ -1910,9 +1920,16 @@ var ConversationView = {
     }
     params.items.push(subjectItem);
 
-    // If we are on a thread, we can call to SelectMessages
+    // If we are on a thread, we can call to SelectMessages and add recipients
     if (Navigation.isCurrentPanel('thread')) {
       params.items.push({
+        l10nId: 'addRecipients-label',
+        method: () => {
+          this.updateDraft();
+          this.initiateNewMessage({ draft: this.draft });
```

Comment:
note I'm not completely sold on this "save the draft, then reuse the draft to save another draft, and then delete the first one". I'll think of it over the night...